<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>notepad</class>
 <widget class="QMainWindow" name="notepad">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>notepad</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="textEdit">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(238, 225, 193);
font: 10pt &quot;Noto Sans&quot;;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>562</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuNew">
    <property name="title">
     <string>New</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionExport_PDF"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionRedo"/>
    <addaction name="actionUndo"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionSelect"/>
    <addaction name="actionBold"/>
    <addaction name="actionItalic"/>
    <addaction name="actionUnderline"/>
    <addaction name="actionNormal"/>
    <addaction name="separator"/>
    <addaction name="actionColor"/>
    <addaction name="actionBackground_Text_Color"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionChange_Area_Color"/>
   </widget>
   <addaction name="menuNew"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionSave_As"/>
   <addaction name="separator"/>
   <addaction name="actionCut"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste"/>
   <addaction name="actionRedo"/>
   <addaction name="actionUndo"/>
   <addaction name="separator"/>
   <addaction name="actionSelect"/>
   <addaction name="actionBold"/>
   <addaction name="actionItalic"/>
   <addaction name="actionUnderline"/>
   <addaction name="separator"/>
   <addaction name="actionQuit"/>
   <addaction name="separator"/>
   <addaction name="actionAbout"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset>
     <normaloff>../../images/open.png</normaloff>../../images/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset>
     <normaloff>../../images/save.png</normaloff>
     <normalon>images/save.png</normalon>../../images/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="icon">
    <iconset>
     <normaloff>../../images/save_as.png</normaloff>
     <normalon>images/save_as.png</normalon>../../images/save_as.png</iconset>
   </property>
   <property name="text">
    <string>Save as</string>
   </property>
   <property name="shortcut">
    <string>Alt+S</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset>
     <normaloff>../../images/exit.png</normaloff>
     <normalon>images/exit.png</normalon>
     <activeon>images/exit.png</activeon>../../images/exit.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="toolTip">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset>
     <normaloff>../../images/cut.png</normaloff>
     <normalon>images/cut.png</normalon>../../images/cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset>
     <normaloff>../../images/copy.png</normaloff>
     <normalon>images/copy.png</normalon>../../images/copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset>
     <normaloff>../../images/paste.png</normaloff>
     <normalon>../../images/paste.png</normalon>../../images/paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset>
     <normaloff>../../images/edit_redo.png</normaloff>
     <normalon>../../images/edit_redo.png</normalon>../../images/edit_redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset>
     <normaloff>images/edit_undo.png</normaloff>
     <normalon>../../images/edit_undo.png</normalon>images/edit_undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset>
     <normaloff>images/info.png</normaloff>
     <normalon>../../images/info.png</normalon>images/info.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I, F1</string>
   </property>
  </action>
  <action name="actionSelect">
   <property name="icon">
    <iconset>
     <normaloff>images/font.png</normaloff>
     <normalon>../../images/font.png</normalon>images/font.png</iconset>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Down</string>
   </property>
  </action>
  <action name="actionBold">
   <property name="icon">
    <iconset>
     <normaloff>images/bold.png</normaloff>
     <normalon>../../images/bold.png</normalon>images/bold.png</iconset>
   </property>
   <property name="text">
    <string>Bold</string>
   </property>
   <property name="shortcut">
    <string>Alt+B</string>
   </property>
  </action>
  <action name="actionItalic">
   <property name="icon">
    <iconset>
     <normaloff>images/italic.png</normaloff>
     <normalon>../../images/italic.png</normalon>images/italic.png</iconset>
   </property>
   <property name="text">
    <string>Italic</string>
   </property>
   <property name="shortcut">
    <string>Alt+I</string>
   </property>
  </action>
  <action name="actionUnderline">
   <property name="icon">
    <iconset>
     <normaloff>images/underline.png</normaloff>
     <normalon>../../images/underline.png</normalon>images/underline.png</iconset>
   </property>
   <property name="text">
    <string>Underline</string>
   </property>
   <property name="shortcut">
    <string>Alt+U</string>
   </property>
  </action>
  <action name="actionNormal">
   <property name="text">
    <string>Normal</string>
   </property>
   <property name="shortcut">
    <string>Alt+N</string>
   </property>
  </action>
  <action name="actionColor">
   <property name="text">
    <string>Color</string>
   </property>
  </action>
  <action name="actionBackground_Text_Color">
   <property name="text">
    <string>Background Text Color</string>
   </property>
  </action>
  <action name="actionBackground_Color">
   <property name="text">
    <string>Background Color</string>
   </property>
  </action>
  <action name="actionChange_Area_Color">
   <property name="text">
    <string>Change Area  Color</string>
   </property>
  </action>
  <action name="actionExport_PDF">
   <property name="text">
    <string>Export PDF</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
